tox.ini

```diff
@@ -52,6 +57,41 @@ commands =
         -n -b {posargs:html} docs/ docs/_build/{posargs:html}
 
 
+[testenv:make-changelog]
```

Comment:
You can use this command to preview the changelog draft on CLI or actually prepare the next release changelog entry if you pass an explicit version as mentioned in the description.